<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>client_window</class>
    <widget class="QMainWindow" name="client_window">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>648</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Messanger Client</string>
        </property>
        <widget class="QWidget" name="messanger_window">
            <property name="minimumSize">
                <size>
                    <width>648</width>
                    <height>600</height>
                </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_2">
                        <item>
                            <widget class="QListWidget" name="dialogs_list">
                                <property name="sizePolicy">
                                    <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                    </sizepolicy>
                                </property>
                                <property name="maximumSize">
                                    <size>
                                        <width>150</width>
                                        <height>16777215</height>
                                    </size>
                                </property>
                                <property name="lineWidth">
                                    <number>1</number>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <layout class="QVBoxLayout" name="verticalLayout">
                                <item>
                                    <widget class="QListWidget" name="messanges_list">
                                        <property name="lineWidth">
                                            <number>2</number>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="Line" name="line_4">
                                        <property name="orientation">
                                            <enum>Qt::Horizontal</enum>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <layout class="QHBoxLayout" name="horizontalLayout_4">
                                        <item>
                                            <widget class="QToolButton" name="tb_b">
                                                <property name="text">
                                                    <string>...</string>
                                                </property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QToolButton" name="tb_i">
                                                <property name="text">
                                                    <string>...</string>
                                                </property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QToolButton" name="tb_u">
                                                <property name="text">
                                                    <string>...</string>
                                                </property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="Line" name="line_2">
                                                <property name="orientation">
                                                    <enum>Qt::Vertical</enum>
                                                </property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QToolButton" name="tb_smile_1">
                                                <property name="text">
                                                    <string>...</string>
                                                </property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QToolButton" name="tb_smile_2">
                                                <property name="text">
                                                    <string>...</string>
                                                </property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QToolButton" name="tb_smile_3">
                                                <property name="text">
                                                    <string>...</string>
                                                </property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QToolButton" name="tb_smile_4">
                                                <property name="text">
                                                    <string>...</string>
                                                </property>
                                            </widget>
                                        </item>
                                        <item>
                                            <spacer name="horizontalSpacer">
                                                <property name="orientation">
                                                    <enum>Qt::Horizontal</enum>
                                                </property>
                                                <property name="sizeHint" stdset="0">
                                                    <size>
                                                        <width>40</width>
                                                        <height>20</height>
                                                    </size>
                                                </property>
                                            </spacer>
                                        </item>
                                    </layout>
                                </item>
                                <item>
                                    <widget class="Line" name="line_3">
                                        <property name="orientation">
                                            <enum>Qt::Horizontal</enum>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <layout class="QHBoxLayout" name="horizontalLayout">
                                        <property name="spacing">
                                            <number>6</number>
                                        </property>
                                        <item>
                                            <widget class="Line" name="line">
                                                <property name="orientation">
                                                    <enum>Qt::Horizontal</enum>
                                                </property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QTextEdit" name="messanger_edit">
                                                <property name="maximumSize">
                                                    <size>
                                                        <width>16777215</width>
                                                        <height>50</height>
                                                    </size>
                                                </property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QPushButton" name="send_button">
                                                <property name="minimumSize">
                                                    <size>
                                                        <width>50</width>
                                                        <height>0</height>
                                                    </size>
                                                </property>
                                                <property name="maximumSize">
                                                    <size>
                                                        <width>16777215</width>
                                                        <height>50</height>
                                                    </size>
                                                </property>
                                                <property name="text">
                                                    <string>Отправить</string>
                                                </property>
                                                <property name="icon">
                                                    <iconset>
                                                        <normaloff>../imgs/send.png</normaloff>../imgs/send.png</iconset>
                                                </property>
                                            </widget>
                                        </item>
                                    </layout>
                                </item>
                            </layout>
                        </item>
                    </layout>
                </item>
            </layout>
        </widget>
        <action name="actionsend">
            <property name="icon">
                <iconset>
                    <normaloff>../imgs/send.png</normaloff>../imgs/send.png</iconset>
            </property>
            <property name="text">
                <string>send</string>
            </property>
        </action>
    </widget>
    <resources/>
    <connections/>
</ui>